tests/e2e-evm/test/extend.ts

```diff
@@ -0,0 +1,52 @@
+import { Address, defineChain, Chain, PublicClientConfig, WalletClientConfig } from "viem";
+import { HardhatRuntimeEnvironment } from "hardhat/types/runtime";
+
+// defaultPublicClientConfig sets default values for viem public client configuration
+const defaultPublicClientConfig: Partial<PublicClientConfig> = {
+  pollingInterval: 100,
+};
+
+// kavalocalnet defines the viem Chain configuration for a kvtool chain
+const kavalocalnet: Chain = defineChain({
+  id: 8888,
+  name: "Kava EVM Localnet",
+  network: "kava-localnet",
+  nativeCurrency: {
+    name: "Kava",
+    decimals: 18,
+    symbol: "KAVA",
+  },
+  rpcUrls: {
+    default: {
+      http: ["http://127.0.0.1:8545"],
+      webSocket: ["ws://127.0.0.1:8546"],
+    },
+  },
+  blockExplorers: undefined,
+  contracts: {},
+});
+
+// getChainConfig returns the kvtoollocalnet Chain if the hardhat environment kvtool network is set, else undefined
+function getChainConfig(hre: HardhatRuntimeEnvironment): { chain?: Chain } {
+  if (hre.network.name == "kvtool") {
```

Comment:
semantics here but if you want the extra type safety ensuring both are strings you want `===`. Probably has no effect here if you know `hre.network.name` always returns a string.

Reply:
would recommend `eqeqeq` linter option to make this decision for us

personal plug for `"smart"` 😄 
https://eslint.org/docs/latest/rules/eqeqeq#smart

Reply:
Fixed in future commit